recipes/cygwin/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: cygwin
+  version: "2.4.1"    # this is the version of the cygwin dll.  Not sure how relevant it is.
+
+# don't download source here - seems there might be some redirect that leads to a different file.
```

Comment:
If this were a more general problem, yes.  However, for one package out of hundreds (many using sourceforge URLs that redirect), I don't want to investigate right now.